app/src/main/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreen.kt

```diff
@@ -11,6 +11,7 @@ import io.reactivex.rxkotlin.ofType
 import org.simple.clinic.ReportAnalyticsEvents
```

Comment:
We should remove the unused imports here.

Reply:
This is also removed in the last commit where we remove the interface. Please check